profile_report.py

```diff
@@ -0,0 +1,31 @@
+from profiler import data
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("benchmark", type=str)
```

Comment:
Would rename `benchmark` to `command`, if only to keep the terminology consistent with the rest of the Profiler code. Consider the signature: `FileBaseProfileDB.find(command: str)`

Reply:
Sounds good